dockertest/subtest.py

```diff
@@ -291,6 +296,22 @@ def make_subsubtest_config(self, all_configs, parent_config,
         self.config = self.make_config(all_configs, parent_config,
                                        self.config_section)
 
+    def require_docker_version(self, wanted):
```

Comment:
I think it's not so much an issue now, but only if/when we ever are testing multiple docker versions installed at the same time.  When that day comes, we can worry about this problem :smile: Until then, ``config["docker_path"]`` isn't all that critical IMHO.